### ISSUE TITLE
issue with newest version of bw cli

### ISSUE THREAD

**Issue by StevenMassaro** [StevenMassaro/email#72]
```2022-12-07T14:42:50.949Z ERROR 1 --- [         task-1] email.service.SyncService                : Exception while processing account .

java.util.concurrent.ExecutionException: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('(' (code 40)): expected a valid value (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"(node:85) UnhandledPromiseRejectionWarning: Error: error:25066067:DSO support routines:dlfcn_load:could not load the shared library
    at Object.privateDecrypt (internal/crypto/cipher.js:63:12)
    at NodeCryptoFunctionService.rsaDecrypt (/usr/local/lib/node_modules/@bitwarden/cli/build/bw.js:32583:58)
    at CryptoService.<anonymous> (/usr/local/lib/node_modules/@bitwarden/cli/build/bw.js:14348:47)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/@bitwarden/cli"[truncated 651 chars]; line: 1, column: 2]
        at com.google.common.util.concurrent.AbstractFuture.getDoneValue(AbstractFuture.java:588) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.util.concurrent.AbstractFuture.get(AbstractFuture.java:547) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.util.concurrent.AbstractFuture$TrustedFuture.get(AbstractFuture.java:113) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.util.concurrent.Uninterruptibles.getUninterruptibly(Uninterruptibles.java:240) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$Segment.getAndRecordStats(LocalCache.java:2317) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2283) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2159) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache.get(LocalCache.java:3966) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$LocalManualCache.get(LocalCache.java:4863) ~[guava-31.1-jre.jar!/:na]
        at email.service.BitwardenService.getLogin(BitwardenService.java:54) ~[classes!/:2.7.3]
        at email.service.SyncService.sync(SyncService.java:41) ~[classes!/:2.7.3]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:na]
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[na:na]
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:na]
        at java.base/java.lang.reflect.Method.invoke(Method.java:568) ~[na:na]
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343) ~[spring-aop-6.0.2.jar!/:6.0.2]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.0.2.jar!/:6.0.2]
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.0.2.jar!/:6.0.2]
        at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.proceed(CglibAopProxy.java:752) ~[spring-aop-6.0.2.jar!/:6.0.2]
        at org.springframework.aop.interceptor.AsyncExecutionInterceptor.lambda$invoke$0(AsyncExecutionInterceptor.java:115) ~[spring-aop-6.0.2.jar!/:6.0.2]
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[na:na]
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[na:na]
        at java.base/java.lang.Thread.run(Thread.java:833) ~[na:na]
Caused by: com.fasterxml.jackson.core.JsonParseException: Unexpected character ('(' (code 40)): expected a valid value (JSON String, Number, Array, Object or token 'null', 'true' or 'false')
 at [Source: (String)"(node:85) UnhandledPromiseRejectionWarning: Error: error:25066067:DSO support routines:dlfcn_load:could not load the shared library
    at Object.privateDecrypt (internal/crypto/cipher.js:63:12)
    at NodeCryptoFunctionService.rsaDecrypt (/usr/local/lib/node_modules/@bitwarden/cli/build/bw.js:32583:58)
    at CryptoService.<anonymous> (/usr/local/lib/node_modules/@bitwarden/cli/build/bw.js:14348:47)
    at Generator.next (<anonymous>)
    at fulfilled (/usr/local/lib/node_modules/@bitwarden/cli"[truncated 651 chars]; line: 1, column: 2]
        at com.fasterxml.jackson.core.JsonParser._constructError(JsonParser.java:2418) ~[jackson-core-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportError(ParserMinimalBase.java:749) ~[jackson-core-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.core.base.ParserMinimalBase._reportUnexpectedChar(ParserMinimalBase.java:673) ~[jackson-core-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.core.json.ReaderBasedJsonParser._handleOddValue(ReaderBasedJsonParser.java:2082) ~[jackson-core-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.core.json.ReaderBasedJsonParser.nextToken(ReaderBasedJsonParser.java:805) ~[jackson-core-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.databind.ObjectMapper._initForReading(ObjectMapper.java:4817) ~[jackson-databind-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4723) ~[jackson-databind-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3677) ~[jackson-databind-2.14.1.jar!/:2.14.1]
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3660) ~[jackson-databind-2.14.1.jar!/:2.14.1]
        at email.service.BitwardenService.deserializeBitwardenJson(BitwardenService.java:75) ~[classes!/:2.7.3]
        at email.service.BitwardenService.lambda$getLogin$0(BitwardenService.java:65) ~[classes!/:2.7.3]
        at com.google.common.cache.LocalCache$LocalManualCache$1.load(LocalCache.java:4868) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3533) ~[guava-31.1-jre.jar!/:na]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2282) ~[guava-31.1-jre.jar!/:na]
```

**Comment by StevenMassaro** [comment]
https://github.com/bitwarden/clients/issues/4196

**Comment by StevenMassaro** [comment]
Upgrade Bitwarden PR: https://github.com/StevenMassaro/email/pull/81

**Comment by StevenMassaro** [comment]
resolved in 61574ef160385eddf98956974bc59d1ba244e9df
